pkg/controller/storagecluster/reconcile.go

```diff
@@ -49,8 +49,10 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 	monCountOverrideEnvVar = "MON_COUNT_OVERRIDE"
 	// EBS represents AWS EBS provisioner for StorageClass
 	EBS StorageClassProvisionerType = "kubernetes.io/aws-ebs"
-	//Name of MetadataPVCTemplate
+	// Name of MetadataPVCTemplate
```

Comment:
Move to the formatting commit.

Reply:
Thanks, required changes have been made!